packages/core/test/io/node-io.test.ts

```diff
@@ -44,6 +55,36 @@ test('@gltf-transform/core::io | node.js read gltf', { skip: environment !== Env
 	t.end();
 });
 
+test('@gltf-transform/core::io | node.js read glb http', { skip: environment !== Environment.NODE }, async (t) => {
+	let count = 0;
+	glob.sync(path.join(__dirname, '../in/**/*.glb')).forEach((inputURI) => {
+		const basepath = inputURI.replace(path.join(__dirname, '../in'), 'http://' + ip);
+
+		const io = new NodeIO(require('node-fetch'));
+		const doc = io.read(basepath);
```

Comment:
Because `io.read` is now asynchronous, it return a promise — this might be throwing errors later, after the test has completed and purportedly passed. One pattern you could use here would be:

```js
	const uris = glob.sync(path.join(__dirname, '../in/**/*.glb'));
	await Promise.all(
		uris.map(async (inputURI) => {
			const io = new NodeIO();
			const doc = await io.read(inputURI);
            // ...
		})
	);
```